.github/stale.yml

```diff
@@ -16,10 +16,10 @@ staleLabel: stale
 
 # Comment to post when marking as stale. Set to `false` to disable
 markComment: >
-  This pull request has been automatically marked as stale because it has not had
-  recent activity. It will be closed in 7 days if no further activity occurs. Please
-  let us know if you are currently working on this PR or have any issues.
-  Thank you for your contributions.
+  Hi @author, I'm going to mark this PR as stale because it hasn't had any updates for 10 days.
+  If no further activity occurs within **7 days**, it will be automatically closed so that others can take up the issue.
+
+  If you don't want this PR to be deleted, please make a follow-up commit within **7 days** (and submit it for review, if applicable). Thanks!
```

Comment:
`If you are still working on this PR, please make a follow-up commit within **7 days**. Please also let us know if you are stuck so we can help you! `

Reply:
Done, thanks!